content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/count_products.rst

```diff
@@ -129,12 +129,16 @@ Count products
 ==============
 
 Counting products is a recurring activity in a warehouse. Once a count is complete, go to
-:menuselection:`Inventory app --> Operations --> Inventory Adjustments` to update the
+:menuselection:`Inventory app --> Operations --> Physical Inventory` to update the
 :guilabel:`Counted Quantity` column for each product line.
 
+To print a PDF of a count sheet, add products to the count sheet by ticking their checkboxes. Click
+:guilabel:`Count sheet` to download a PDF that contains the selected products.
+
 On each product line, identify whether the value in the :guilabel:`On Hand Quantity` column recorded
 in the database matches the newly-counted value. If the recorded value and the counted value do
-match, click the :guilabel:`Set` button (target icon) at the far right of the product line.
+match, select the product using the checkbox, and then select
+:menuselection:`Actions --> Set to quantity on hand`.
```

Comment:
This is our team's specific style, but we only use the menuselection tag when navigating menus 
![image](https://github.com/user-attachments/assets/30680b5e-4a37-417c-b3fd-5708472efb15)

Actions is a button that has "drop-down menu options". Very minor distinction, I know. :see_no_evil: 

```suggestion
the :guilabel:`Actions` button, then :guilabel:`Set to quantity on hand`.
```